### PR TITLE
add nisqAnalyzerQpuSelection feature

### PR DESCRIPTION
**Pull request by salmma**
Add nisqAnalyzerQpuSelection feature to support qpu selection ui feature, see https://github.com/UST-QuAntiL/qc-atlas-ui/pull/139.
On default, only the qpu selection feature is visible.
_We may want to set UI for PatternAtlas, QProv, and showSettings also to false?_

### CODE REVIEW

**Comment by salmma** [comment]
**NOTE:** Updated components need new versions!

File: PLATFORM_NISQ_README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+This guidance guides you through the [QC Atlas UI](https://github.com/UST-QuAntiL/qc-atlas-ui) for applying the [NISQ Analyzer](https://github.com/UST-QuAntiL/nisq-analyzer) on given implementations of the PlanQK Platform for selecting suitable quantum computers.  
+
+### Prerequisites
++ Docker Engine - [Install Guide](https://docs.docker.com/engine/install/)
++ Docker Compose - [Install Guide](https://docs.docker.com/compose/install/)
++ Clone [quantil-docker repository](https://github.com/UST-QuAntiL/quantil-docker)
++ Required ports are listed [here](https://github.com/UST-QuAntiL/quantil-docker#readme)
+
+### Prerequisites for the NISQ Analyzer and the given Implementation
+For using the NISQ Analyzer, currently, on the platform itself, the field _technology_ specifies the SDK (e.g. Qiskit) the implementation was written with and the field _version_ specifies the programming language (e.g. Qiskit/OpenQASM).
```
Comment (lharzenetter) [inline PLATFORM_NISQ_README.md]: To use the NISQ Analyzer on the PlanQK platform content, currenly only Qiskit-based implementations are supported.
Thus the _technology_ field is used to specify the used SDK, i.e., Qiskit, while the _version_ field spcifies the programming langues used, e.g., Qiskit or OpenQASM.

File: PLATFORM_NISQ_README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Also note that only the first file on the platform is currently used by the NISQ Analyzer for analyzing and executing the implementation.
```
Comment (lharzenetter) [inline PLATFORM_NISQ_README.md]: Please note that currently only ....

File: PLATFORM_NISQ_README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Further, currently, only fix circuits without input parameters can be executed, see, e.g., the implementation of Shor.
```
Comment (lharzenetter) [inline PLATFORM_NISQ_README.md]: A current limitation is that onlz fixed cicruits, i.e., circuits that do not require input parameters, can be executed. The Shor implementation is such an example.

File: PLATFORM_NISQ_README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+### 1. Add Qiskit Credentials to docker-compose.yml
+As currently only quantum computers of IBMQ are supported, your Qiskit token has to be inserted into the docker-compose.yml ([here](https://github.com/UST-QuAntiL/quantil-docker/blob/06232eb6520a83333405d8c83d85eace27f96a37/docker-compose.yml#L271)) file for accessing them:
```
Comment (lharzenetter) [inline PLATFORM_NISQ_README.md]: ~As~ Currently, ... supported. Thus, you must insert your Qiskit token ...

Comment (lharzenetter) [reply PLATFORM_NISQ_README.md]: I would add a `_docker-compose.override` file with example overrides and users must copy it to a `docker-compose.override` file (which is ignored by git)  where the token should be added.
Also remove the toke element from the normal docker-compose file.

**Review by lharzenetter — CHANGES_REQUESTED** [review]

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**